pkg/operator/apiserver/controller/apiservice/apiservice_controller.go

```diff
@@ -23,13 +24,16 @@ import (
 	"github.com/openshift/library-go/pkg/operator/v1helpers"
 )
 
-type GetAPIServicesToMangeFunc func() ([]*apiregistrationv1.APIService, error)
+// GetAPIServicesToMangeFunc provides a two list of managed APIService items.
+// Both lists need to always contain all the managed APIServices so only managed
+// objects are removed without removing user-created/unmanaged objects.
```

Comment:
what do you mean by `without removing user-created/unmanaged objects` ?

Reply:
Comment updated.